src/Components/Card/Card.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReservationBtn from '../Reservation/ReservationBtn';
+import './card.css';
 
 const Card = ({
   img, title, desc, rocketId, reserved,
 }) => (
-  <article>
-    <img src={img} alt="" />
-    <div>
-      <h4>{title}</h4>
+  <article className="card">
+    <img className="image" src={img} alt="" />
+    <div className="card-desc">
+      <h4 className="title">{title}</h4>
       <p>
-        <span>{reserved ? 'Reserved' : ''}</span>
+        {reserved ? <span className="status">Reserved</span> : ''}
```

Comment:
Great change! You could even go further by using the `&&` operator, like this: 

```js
{reserved && <span className='status'> Reserved </span>}
```

This way, only when the boolean 'reserved' is true, is going to render the span.